lighthouse/src/main/java/com/ivanempire/lighthouse/models/search/MulticastSearchRequest.kt

```diff
@@ -48,17 +47,19 @@ data class MulticastSearchRequest(
     override fun toString(): String {
         val builder = StringBuilder()
         builder.append(StartLine.SEARCH.rawString).append(NEWLINE_SEPARATOR)
-        builder.append(HeaderKeys.HOST).append(FIELD_SEPARATOR).append(hostname.toString()).append(NEWLINE_SEPARATOR)
-            .append(HeaderKeys.MAN).append(FIELD_SEPARATOR).append(DEFAULT_SEARCH_MAN).append(NEWLINE_SEPARATOR)
-            .append(HeaderKeys.MX).append(FIELD_SEPARATOR).append(mx).append(NEWLINE_SEPARATOR)
-            .append(HeaderKeys.SEARCH_TARGET).append(FIELD_SEPARATOR).append(searchTarget).append(NEWLINE_SEPARATOR)
+        builder.append(HeaderKeys.HOST).append(FIELD_SEPARATOR).append(" ").append(hostname.toString()).append(NEWLINE_SEPARATOR)
+            .append(HeaderKeys.MAN).append(FIELD_SEPARATOR).append(" ").append(DEFAULT_SEARCH_MAN).append(NEWLINE_SEPARATOR)
```

Comment:
Do some of the devices ignore the message if there's no space? Asking because I didn't find any requirements in the UPnP spec on this, at least not in the M-SEARCH message section?

Reply:
They are defined as HTTP 1.1 headers. https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

There's no requirement for spaces, but all of the examples in the UPnP spec have spaces, so I added them for an abundance of caution. Actually, I added them when my camera wasn't being recognized and that was one of the differences from a message that did prompt a response. The actual issue was the missing blank line, but I think the spaces are more likely to do good than harm.

Reply:
I'm going to cherrypick this commit :)